spec/models/requests/requestable_decorator_spec.rb

```diff
@@ -1482,6 +1497,13 @@
       end
     end
 
+    context "at recap - firestone$pf" do
+      let(:stubbed_questions) { default_stubbed_questions.merge(recap?: true, recap_pf?: true, library_code: 'firestone$pf', holding_library: 'firestone') }
```

Comment:
Is this correct `holding_library: 'firestone'` @kevinreiss

Reply:
Based on https://bibdata.princeton.edu/locations/holding_locations/firestone$pf I think it is .